packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1733,11 +1748,30 @@ describe("calcite-combobox", () => {
       const button = await page.find("button");
       await input.click();
       await input.press("o");
+
+      await page.waitForChanges();
+      await page.waitForTimeout(DEBOUNCE.filter);
+
       await input.press("Tab");
+
+      await page.waitForChanges();
+      await page.waitForTimeout(DEBOUNCE.filter);
```

Comment:
As a follow-up, let's see if this extra pair of `waitForX`s are needed. Also, some of the E2E/Puppeteer APIs call `waitForChanges` under the scenes, so some calls may not be necessary.

Reply:
Sounds good, I'm removing that waitForX double, seem so to pass without it.